chain/client/src/info.rs

```diff
@@ -142,10 +142,14 @@ impl InfoHelper {
     }
 
     /// Count which shards are tracked by the node in the epoch indicated by head parameter.
```

Comment:
The comment seems totally unrelated to what this method does. Can you fix it?

Reply:
done